src/storage.cc

```diff
@@ -209,6 +212,9 @@ Status Storage::Open(bool read_only) {
   metadata_opts.table_factory.reset(rocksdb::NewBlockBasedTableFactory(metadata_table_opts));
   metadata_opts.compaction_filter_factory = std::make_shared<MetadataFilterFactory>(this);
   metadata_opts.disable_auto_compactions = config_->RocksDB.disable_auto_compactions;
+  // Enable whole key bloom filter in memtable
+  metadata_opts.memtable_whole_key_filtering = true;
+  metadata_opts.memtable_prefix_bloom_size_ratio = 0.1;
```

Comment:
cool, does it cause write performance loss? is there a read improvement?

Reply:
I did a simple test and the performance didn't drop, but there was no significant improvement either; 
In theory, this operation can reduce CPU utilization, which is also beneficial.